### PR TITLE
Starting Timeline Server in JOBHISTORY to fix job loss issue

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

**Comment by yqwang-ms** [comment]
wait AHS tested ok

**Comment by yqwang-ms** [comment]
[http://10.151.40.234:8188/applicationhistory](http://10.151.40.234:8188/applicationhistory)

---
In reply to: [382681010](https://github.com/Microsoft/pai/pull/503#issuecomment-382681010) [](ancestors = 382681010)

**Comment by fanyangCS** [comment]
framework launcher unit test failed due to permission denied. @Yuqi Wang, can you take a look?

---
In reply to: [382681351](https://github.com/Microsoft/pai/pull/503#issuecomment-382681351) [](ancestors = 382681351,382681010)

File: service-deployment/config-patch/nodemanager-mapred-site.xml.patch [hunk 1]
```diff
@@ -6,7 +6,7 @@
  <!--
    Licensed under the Apache License, Version 2.0 (the "License");
    you may not use this file except in compliance with the License.
-@@ -14,8 +13,22 @@
+@@ -14,8 +13,27 @@
```
Comment (hwuu) [inline service-deployment/config-patch/nodemanager-mapred-site.xml.patch]: I’m curious about why we need mapred. Could you explain in brief?

File: service-deployment/bootstrap/hadoop-service/hadoop-data-node.yaml.template [hunk 2]
```diff
@@ -40,6 +40,8 @@ spec:
           name: hadoop-config-volume
         - mountPath: /host-configuration
           name: host-confg-volume
+        - mountPath: /var/lib/hadoopdata
+          name: hadoop-tmp-storage
```
Comment (hwuu) [inline service-deployment/bootstrap/hadoop-service/hadoop-data-node.yaml.template]: Could you explain what this folder is created for?

**Pull request merged**